config/default_records/plate_purposes/015_rna_stock_plate.wip.yml

```diff
@@ -0,0 +1,6 @@
+---
+Stock RNA Plate:
```

Comment:
possibly add `input_plate: false` to be clear it's not an input, but I think it's the default so not strictly required